README.md

```diff
@@ -59,7 +60,8 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+1. expression: weather is assgined to sunny.
+2. expression: is checking if the type and the value of weather are equal, in this case : true
```

Comment:
Actually, each line is an expression!

Line 1, `var weather;`, evaluates to `undefined`.
Line 2, `weather = "sunny"`, evaluates to `"sunny"` because the `=` operator always evaluates to the value that's being assigned.
Line 3, `weather === "sunny"`, evaluates to `true`.